src/basic.rs

```diff
@@ -225,7 +227,9 @@ impl convert::From<parquet::CompressionCodec> for Compression {
       parquet::CompressionCodec::SNAPPY => Compression::SNAPPY,
       parquet::CompressionCodec::GZIP => Compression::GZIP,
       parquet::CompressionCodec::LZO => Compression::LZO,
-      parquet::CompressionCodec::BROTLI => Compression::BROTLI
+      parquet::CompressionCodec::BROTLI => Compression::BROTLI,
+      parquet::CompressionCodec::LZ4 => Compression::LZ4,
```

Comment:
You may need to add a few trivial tests in `test_display_compression` and `test_from_compression`.

Reply:
Will add, thanks.